swift/integration-tests/hello-world/.gitignore

```diff
@@ -0,0 +1,7 @@
+.DS_Store
+/.build
+/Packages
+/*.xcodeproj
+xcuserdata/
+DerivedData/
+.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
```

Comment:
would it make sense to move some of these up in a `integration-tests/.gitignore` file?

Reply:
Right, it was generated by `swift package init`. I'll move it.

Reply:
@redsun82 done.